src/cpp/session/modules/ModuleTools.R

```diff
@@ -456,3 +456,8 @@ if (getRversion() < "3.5") {
       }
    })
 }
+
+.rs.addFunction("sessionModulePath", function() {
+   .Call("rs_sessionModulePath")
```

Comment:
I think we should enforce using `PACKAGE = "(embedding)"` on any new `.Call()` usages, as otherwise you're asking R to crawl through all of the R DLLs to find something called `rs_sessionModulePath` and IIUC the RStudio DLL can be on the tail end of the search list.